app/templates/auth/host/greeting.hbs

```diff
@@ -19,7 +19,13 @@ as |page|
           data-test-profile-pic=true
         }}
       {{/form.group}}
-
+      {{#form.group label="Profile Name" as |group|}}
+        {{group.input
```

Comment:
Similar question here -- I think we wanted the profile name to be above the user's name on the host profile page? Or to replace it? I don't quite recall, but we should sort that out.

Reply:
Will update the preview-page later today. And yes, I remembered the contact person's will be replaced by the profile name and will do accordingly.

Reply:
Revised